### PR TITLE
[Docs] Adding region support, and emissary changes

### PR DESCRIPTION
**Pull request by jonahjon**
Signed-off-by: jonahjon <jonahjones094@gmail.com>

- Adding support for different regions for EKS-A packages.
- Adding new Emissary version, and note about multiple instances of packages.

### CODE REVIEW

**Comment by ivyostosh** [comment]
For emissary new version, could you update [this index file](https://github.com/aws/eks-anywhere/blob/main/docs/content/en/docs/reference/packagespec/_index.md) as well?

Everything else looks good to me!

**Comment by ivyostosh** [comment]
/lgtm
you can decide when to merge it :)

File: docs/content/en/docs/tasks/troubleshoot/packages.md [hunk 1]
```diff
@@ -131,7 +131,7 @@ If the image downloads successfully, it worked!
 You may need to create or update your credentials which you can do with a command like this. Set the environment variables to the proper values before running the command.
 ```bash
 kubectl delete secret -n eksa-packages aws-secret
-kubectl create secret -n eksa-packages generic aws-secret --from-literal=ID=${EKSA_AWS_ACCESS_KEY_ID} --from-literal=SECRET=${EKSA_AWS_SECRET_ACCESS_KEY}
+kubectl create secret -n eksa-packages generic aws-secret --from-literal=ID=${EKSA_AWS_ACCESS_KEY_ID} --from-literal=SECRET=${EKSA_AWS_SECRET_ACCESS_KEY}  --from-literal=SECRET=${EKSA_AWS_REGION}
```
Comment (vincentni) [inline docs/content/en/docs/tasks/troubleshoot/packages.md:134]: It should be `--from-literal=REGION=${EKSA_AWS_REGION}`

**Pull request merged**